code/index.html

```diff
@@ -2,21 +2,92 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>classic newspaper</title>
     <link rel="stylesheet" href="./style.css" />
+    <meta name="viewport" content="width=device, initial-scale=1.0">
     <!-- dont forget the meta tag for responsiveness -->
   </head>
+ 
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+    <header>        
+       <section class="header-container" >
+          <img src="divider-4715969.svg" alt="devider">
+          <h1>Classic Newspaper</h1>
+        </section>
+        <section class="the-first-container">
+          <div class="hamburger"> 
+            <div></div>
+            <div></div>
+            <div></div>
+          </div>
+          <nav class=nav>
+            <a href="#" ></a>World</a>
+            <a href="#" ></a>Bussines</a>
+            <a href="#" ></a>Politics</a>
+            <a href="#" ></a>Sport</a>
+          <nav>
+         </section> 
+     </header>
 
+<main>
+   <!--BIG NEWS!--> 
+  <section>
     <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
+        <Section class="text">
+          <article> <h2>Titanic sinks four hours after hitting iceberg</h2></article> 
+          <article> <p>The Titanic started her trip from Southampton for New York on Wednesday. Late on Sunday night she struck an iceberg off the Grand Banks of Newfoundland. By wireless telegraphy she sent out signals of distress, and several liners were near enough to catch and respond to the call. <br> Conflicting news, alarming and reassuring, was current yesterday. Even after midnight it was said all the passengers were safe. All reports, of course, depended on wireless telegrams over great distances. <br> Late last night the White Star officials in New York announced that a message had been received stating that the Titanic sank at 2.20 yesterday morning after all her passengers and crew had been transferred to another vessel. Later they admitted that many lives had been lost. An unofficial message from Cape Race, Newfoundland, stated that only 675 have been saved out of 2,200 to 2,400 persons on board. </p></article>
```

Comment:
If this is the same article you can have everything related inside one `<article>` element